exercises/concept/train-driver/.docs/instructions.md

```diff
@@ -0,0 +1,111 @@
+# Instructions
+
+Your friend is a train driver and has to drive cargo trains between cities.
+Although your friend isn't amazing with handling the computers and would like some help with it.
+Your friend would like your help organizing the train and correcting the mistakes in the data.
+
+
+```exercism/note
+To practice, use a for the rest or spread to solve each of the tasks below.
+```
+
+## 1. Convert the data to an array
+
+Your friend has been keeping track of how much each wagon weighs. Although they are not sure how many wagons and would like the data to be returned as an array.
+
```

Comment:
I find the story here a bit confusing, e.g. the name of the function does not reflect that it is actually about weights. What about saying that each wagon has a number as identifier and the "engine wagon" (not sure how it is called in English) at the front has number 1. The task then can be to take the wagon numbers as arguments and return the list as array including the engine wagon. The function could be called "addWagons".

If you go with this, other tasks should be adjusted accordingly. E.g. you could say in task 2 that 2 wagons arrived late and are now at the end instead of at the expected place in the front.

Reply:
Locomotive is the engine wagon called

Great idea!